frontend/src/views/admin/assignments/offer-confirmation-dialog.tsx

```diff
@@ -107,11 +109,11 @@ export function MultiWithdrawOfferConfirmation(props: {
                 </Button>
                 <Button
                     onClick={() => {
-                        withdrawOffers();
+                        callback();
```

Comment:
We should await the callback and set a spinner state while the callback is executing. The dialog shouldn't close until the callback is finished. You can look at some of the other dialogs for an example of this.